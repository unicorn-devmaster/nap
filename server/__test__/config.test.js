/* eslint-env jest */

process.env.MAILGUN_API_KEY = 'FOO_MAILGUN_API_KEY'
process.env.MAILGUN_DOMAIN = 'BAR_MAILGUN_DOMAIN'

const config = require('../config')

describe('config', () => {
  it('should return config', () => {
    expect(config).toMatchSnapshot()
  })
})
