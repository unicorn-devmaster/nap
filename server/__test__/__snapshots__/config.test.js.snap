// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config should return config 1`] = `
Object {
  "cookie_secret": "foo",
  "dev": true,
  "graphiql_enabled": true,
  "graphql_disabled": false,
  "jwt_secret": "foo",
  "mailgun_api_key": "FOO_MAILGUN_API_KEY",
  "mailgun_domain": "BAR_MAILGUN_DOMAIN",
  "mongo_url": "mongodb://mongo/graphql",
  "mubsub": undefined,
  "mubsub_enabled": false,
  "passport_disabled": false,
  "port": 3000,
  "redis_url": "redis://redis",
}
`;
